.github/workflows/build-container.yml

```
@@ -0,0 +1,61 @@
+name: Build container Test
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build-container-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          # git submodule update --init --recursive
+          submodules: recursive
+
+      - name: Build container images
+        run: make build
+
+      - name: Setup huge pages
+        run: make setup
+
+      - name: Compose Up
+        run: |
+          make up || (make logs; exit 1)
+
+      - name: Wait for controller to start
+        run: |
+          until nc -z localhost 5500; do
```

Nit: I also have to replace many hardcoded stuff in my original PR 😁 
```suggestion
          until nc -z localhost $PORT; do
```

---

Extracted PORT as env. variable.

---

Thanks @baum !